deploy/sre-prometheus/100-cloud-ingress-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: monitoring.coreos.com/v1
+￼kind: PrometheusRule
```

Comment:
You have some hidden characters in here that is causing the generated SSS to break below

Reply:
ah thank you for catching that!